docs/source/interfaces/Indirect Settings.rst

```diff
@@ -0,0 +1,336 @@
+
+OK
+  This will apply the changes made to the interface and then exits the settings window.
+
+Cancel
+  This disregards any changes which have not been applyed and exits the window.
+
+
+General
+-------
+
+This section contains options which will be applyed across all of the Indirect interfaces.
+
+Options
+~~~~~~~
+
+Facility
+  Allows you to choose the selected facility.
+
+
+Indirect Bayes
+--------------
+
+This section is displayed only when the settings window is opened from the Bayes
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Analysis
+-------------
+
+This section is displayed only when the settings window is opened from the Data Analysis
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Corrections
+----------------
+
+This section is displayed only when the settings window is opened from the Data Corrections
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Reduction
+--------------
+
+This section is displayed only when the settings window is opened from the Data Reduction
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Other Interfaces
+----------------
+
+There are currently no interface-specific settings for the Diffraction, Simulation and Tools
+interfaces.
+
+
+Glossary of Allowed Suffixes
```

Comment:
Thanks for adding this. This seems to be a type of text that could be best represented as a table. It could have 4 columns: the suffix, the x-axis, the histogram axis (y axis is not needed, I think), and where it is produced.